tensorflow/python/tfe_wrapper.cc

```diff
@@ -369,30 +369,28 @@ PYBIND11_MODULE(_pywrap_tfe, m) {
   });
 
   // // TFE_Context Logic
-  m.def(
-      "TFE_NewContext",
-      [](const TFE_ContextOptions* opts) {
-        tensorflow::Safe_TF_StatusPtr status =
-            tensorflow::make_safe(TF_NewStatus());
-        TFE_Context* context = TFE_NewContext(opts, status.get());
-        tensorflow::MaybeRaiseRegisteredFromTFStatus(status.get());
-        return tensorflow::PyoOrThrow(tensorflow::OutputTFE_Context(context));
-      },
-      py::return_value_policy::reference);
+  m.def("TFE_NewContext",
```

Comment:
A minor nit, but it looks like your editor/code formatter is making lots of unrelated format changes. It makes the diff hard to interpret.